ozaria/site/views/play/level/tome/Problem.coffee

```diff
@@ -37,7 +37,13 @@ module.exports = class Problem
       @createdBy = 'web-dev-iframe'
       # TODO: Include runtime/transpile error types depending on something?
 
-    @message = @translate(@message)
+    camelToSnake = (str) ->
+      str.replace(/[A-Z]/g, (letter) -> "_#{letter.toLowerCase()}");
+    debugger
```

Comment:
```suggestion
```

Reply:
OMG why is there a `debugger` left in the code 😂 💣